runner/src/main.rs

```diff
@@ -45,15 +181,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             run_bazel_build(),
             run_bazel_test(),
         ],
-    };
-    let statuses = run_step(&Context::root(&opt), &root);
-
-    // If the overall status value is an error, terminate with a nonzero exit code.
-    if statuses.contains(&StatusResultValue::Error) {
-        std::process::exit(1);
     }
+}
 
-    Ok(())
+fn examples() -> impl Iterator<Item = PathBuf> {
+    let walker = walkdir::WalkDir::new("./examples").into_iter();
+    walker
+        .filter_entry(|e| !is_ignored_entry(e))
```

Comment:
Does it also ignore `./examples/certs` ?
https://github.com/tiziano88/oak/blob/a8554dbebeeac6bb9e660e0bc20d548c854ccdb9/runner/src/main.rs#L197-L206

Reply:
No, but this is used with extra filtering (only including Cargo.toml files), so in the end that does not affect it.